kit/src/test/resources/com/oracle/javafx/scenebuilder/kit/fxom/HeaderWithNotOnlyImports.fxml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?language javascript?>
+<?language nameOfScriptLanguage?>
```

Comment:
I assume this is a required change related to the recent jfx 20 update

Reply:
Its related to fact that the "FXML JavaScript Engine" is now disabled by default. Hence the related test will fail with an error. However, for the actual test it is not of interest which language is used. Hence for the test its okay to attempt loading the FXML like so. It would have been possible to enable the engine just for this test but this is indeed not needed.

Reference: https://gluonhq.com/products/javafx/openjfx-20-release-notes/